MobileClient/Client/Components/HorizontalProductScroller.razor.cs

```diff
@@ -15,5 +18,19 @@ public partial class HorizontalProductScroller
 
     [Parameter]
     public IEnumerable<Item> Items { get; set; } = default!;
+
+    protected override async Task OnParametersSetAsync()
```

Comment:
Vielleicht sollte das ganze in AfterRender im FirstRender passieren, sodass erstmal alle Items ohne Bild angezeigt werden.
Sobald alle Pictures zugewiesen wurden kann dann StateHasChanged() aufgerufen werden.

Alternative:
Das ganze wird bei jedem nach jedem Render und nicht nur dem ersten aufgerufen und Item die schon ein Bild haben werden geskippt.
Dann jedes mal wenn ein Item ein Bild zugewiesen bekommen hat kann StateHasChanged() aufgerufen werden, dadurch würden die Bilder nach und nach erscheinen.